setup.py

```diff
@@ -79,6 +79,7 @@
     'samp_hub = astropy.vo.samp.hub_script:hub_script',
     'volint = astropy.io.votable.volint:main',
     'wcslint = astropy.wcs.wcslint:main',
+    'apy-affiliated = astropy.utils.affiliated:main'
```

Comment:
I feel like just spelling out "astropy" is fine; less cryptic, more (or at least equally) autocomplete-friendly.